flink-connector-jdbc/pom.xml

```diff
@@ -39,7 +39,7 @@ under the License.
 		<scala-library.version>2.12.7</scala-library.version>
 		<assertj.version>3.23.1</assertj.version>
 		<postgres.version>42.5.1</postgres.version>
-		<oracle.version>19.3.0.0</oracle.version>
+		<oracle.version>21.8.0.0</oracle.version>
```

Comment:
Just to verify; does bumping the Oracle version have any impact for users? Does it mean that we're dropping support for older Oracle versions, any other things that are worthwhile to consider?

Reply:
They are compatible. The dependency is provided so users still have to provide the version they want.. Basically we are testing against the last version (I update the docker as well)..

I could leave the current version and update it on a separated commit if is preferred..